### PR TITLE
Rename cancel to stop in audio preview

### PR DESCRIPTION
**Pull request by mpozy**
Also rename the `RecorderStatus` to `AudioCaptureOptions`.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ version '1.0-SNAPSHOT'
 
 buildscript {
     ext.kotlin_version = '1.7.10'
-    ext.voxeet_sdk_version = "3.10.0"
+    ext.voxeet_sdk_version = "3.10.+"
```
Comment (graduad) [inline android/build.gradle:6]: Could we use the bracket notation so we can specify a lower bound of 3.10.1 ?

File: ios/Classes/Bindings/AudioPreviewServiceBinding.swift [hunk 2]
```diff
@@ -105,11 +105,11 @@ class AudioPreviewServiceBinding: Binding {
     /// - Parameters:
     ///   - flutterArguments: Method arguments passed from Flutter.
     ///   - completionHandler: Call methods on this instance when execution has finished.
-    public func cancel(
+    public func stop(
```
Comment (graduad) [inline ios/Classes/Bindings/AudioPreviewServiceBinding.swift:108]: Please change the documentation as well.

File: test_app/ios/Podfile.lock [hunk 3]
```diff
@@ -41,15 +41,13 @@ PODS:
     - Flutter
   - permission_handler_apple (9.0.4):
     - Flutter
-  - SDWebImage (5.16.0):
-    - SDWebImage/Core (= 5.16.0)
-  - SDWebImage/Core (5.16.0)
+  - SDWebImage (5.17.0):
+    - SDWebImage/Core (= 5.17.0)
+  - SDWebImage/Core (5.17.0)
```
Comment (graduad) [inline test_app/ios/Podfile.lock:46]: Is it possible not to upgrade this?

File: ios/Classes/Bindings/AudioPreviewServiceBinding.swift [hunk 4]
```diff
@@ -101,15 +101,15 @@ class AudioPreviewServiceBinding: Binding {
         }
     }
 
-    /// If playing or recording is underway calling this method will cancel the ongoing activity.
+    /// Stops recording or playing an audio sample. This method is available in SDK 3.10.1 and later.
```
Comment (graduad) [inline ios/Classes/Bindings/AudioPreviewServiceBinding.swift:104]: What about this: 

> If playing or recording is underway calling this method will stop the ongoing activity.

File: test_app/ios/Podfile.lock [hunk 5]
```diff
@@ -32,41 +32,39 @@ PODS:
...
-    - VoxeetSDK (~> 3.10.0)
+    - VoxeetSDK (~> 3.10.1)
   - file_picker (0.0.1):
     - DKImagePickerController/PhotoGallery
     - Flutter
   - Flutter (1.0.0)
   - integration_test (0.0.1):
     - Flutter
-  - permission_handler_apple (9.0.4):
+  - permission_handler_apple (9.1.1):
```
Comment (graduad) [inline test_app/ios/Podfile.lock:42]: Would it be possible not to upgrade this?

**Review by graduad — APPROVED** [review]

**Review by kbetl-dlb — APPROVED** [review]

**Pull request merged**